code/modules/client/preference_setup/general/03_background.dm

```diff
@@ -51,8 +51,8 @@
 /datum/category_item/player_setup_item/general/background/sanitize_character()
 	if(!pref.home_system) pref.home_system = "Vetra"
 	if(!pref.citizenship) pref.citizenship = "Blue Colony"
-	pref.home_system = sanitize_inlist(pref.home_system, home_system_choices, initial(pref.home_system))
-	pref.citizenship = sanitize_inlist(pref.citizenship, citizenship_choices, initial(pref.citizenship))
+//	pref.home_system = sanitize_inlist(pref.home_system, home_system_choices, initial(pref.home_system))
+//	pref.citizenship = sanitize_inlist(pref.citizenship, citizenship_choices, initial(pref.citizenship))
```

Comment:
```suggestion

```

Reply:
If you are sure, you can just remove unused code, not comment it. Commenting code is a anti-pattern and makes many mess in code and must be used only when it really needed (example: disabling feature until bug fixed)